### PR TITLE
Add/Issue-492: Table Embed Instructions

### PR DESCRIPTION
**Pull request by Erin-Cecele**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [X] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

Yes, #492 

### Please describe the goal of this pull request and the changes that were made.

Provide instructions on how to add a table as a .html file. 

### Additional Comments

@geealbers I believe this is incomplete. I would like to add instructions on how to add a table if it is _not_ already  in a Word doc format. Does that mean manually creating the table in Markdown? You also mentioned that when converting the word doc to html there may be "junk" that gets included. Still not sure what you mean by that and want to address it in the docs if it's a possible issue.

### CODE REVIEW

File: content/documentation/figure-images.md [hunk 1]
```diff
@@ -156,6 +156,25 @@ Quire supports video embeds from either YouTube (`media_type: youtube`) or Vimeo
 
 {{< /q-class >}}
 
+## Add Tables
+
+Tables can be embedded in your publication as HTML files. If your table exists in a Word document, resave the Word document as an .html file. In the `static/image/` directory of your project create a new folder called `tables`. Place the .html files in that folder.
```
Comment (geealbers) [inline content/documentation/figure-images.md:161]: Users shouldn't put tables in directly from Word. And I realized in reading this that that's what you did with Cultural Heritage. The two tables in that book were simple enough so as an example I just did them while I was also fixing some of the style issues you say, here's what the second one looks like now:

```html
<table>
  <thead>
    <tr>
      <th>Denomination</th>
      <th>Mosul-Old City</th>
      <th>Nineveh Plain</th>
      <th>Total</th>
    </tr>
  </thead>
  <tbody>
    <tr><td>Sunni</td><td>35</td><td>6</td><td>41</td></tr>
    <tr><td>Shi’ite</td><td>1</td><td>73</td><td>74</td></tr>
    <tr><td>Yazidi</td><td>0</td><td>26</td><td>26</td></tr>
    <tr><td>Christian</td><td>3</td><td>6</td><td>9</td></tr>
    <tr><td>Other/Misc.</td><td>2</td><td>1</td><td>3</td></tr>
    <tr><td>TOTAL</td><td>41</td><td>114</td><td>155</td></tr>
  </tbody>
</table>
```

That's all that's in the `static/img/tables/table-9-2-stein.html` file. No other HTML tags, no styles, not `id`s or other attributes within the HTML. Just a plain `<table>`.

I think the instruction in the Quire docs here should be that they can use Word as a starting point, but might find hand building tables easier? There are probably also tools to help? Maybe Excel tables would be easier to convert? I'm not sure.

File: content/documentation/figure-images.md [hunk 2]
```diff
@@ -156,6 +156,25 @@ Quire supports video embeds from either YouTube (`media_type: youtube`) or Vimeo
...
+- id: "table-1-1"
+  src: "tables/table-1-1.html"
+  media_type: table
+  label: "Table 1.1"
+```
+
+Then use the `q-figure` shortcode as normal to insert the table in to the page:
+
+```go
+{{</* q-figure id="table-1-1" */>}}
```
Comment (geealbers) [inline content/documentation/figure-images.md:175]: Do we also want to mention that users could add simple tables using Markdown's built-in table structuring, but that the tables won't be zoomable like other figures?

**Review by geealbers — COMMENTED** [review]
Good to have this in the docs. I didn't realize we didn't have anything at all in there! But I think it needs a little more info. Maybe we need to point to examples or an HTML tuturial on tables?

File: content/documentation/figure-images.md (hunk 2 above)
Comment (Erin-Cecele) [reply content/documentation/figure-images.md:175]: Ah yes! I think we should definitely.

File: content/documentation/figure-images.md (hunk 1 above)
Comment (Erin-Cecele) [reply content/documentation/figure-images.md:161]: I think examples that people can maybe cut and paste from or that explain the html table structure would be good. 

I also found this resource which is really helpful: https://www.w3schools.com/html/html_tables.asp. Might be nice to include, although I noticed you used slightly different html including <thead> and <tbody>. Is that specific to Quire or just a different way of doing it?

Comment (Erin-Cecele) [reply content/documentation/figure-images.md]: @geealbers Just pushed a new commit where I added an html table example. What do you think?

File: content/documentation/figure-images.md [hunk 3]
```diff
@@ -156,6 +156,60 @@ Quire supports video embeds from either YouTube (`media_type: youtube`) or Vimeo
 
 {{< /q-class >}}
 
+## Add Tables
+
+Zoomable tables can be embedded in your publication as HTML files. In the `static/image/` directory of your project create a new folder called `tables`. Create an .html file in that folder that follows the structure below:
+
+``` go
```
Comment (geealbers) [inline content/documentation/figure-images.md:163]: Should probably be `html` instead of `go` to ensure proper syntax highlighting

**Review by geealbers — APPROVED** [review]

File: content/documentation/figure-images.md (hunk 3 above)
Comment (Erin-Cecele) [reply content/documentation/figure-images.md:163]: Is this what you mean?

Screen Shot 2022-01-25 at 12 22 01 PM

Comment (geealbers) [reply content/documentation/figure-images.md:163]: yep!

**Pull request merged**